uberfire-extensions/uberfire-wires/uberfire-wires-core/uberfire-wires-core-grids/src/test/java/org/uberfire/ext/wires/core/grids/client/util/CellContextUtilitiesTest.java

```diff
@@ -178,18 +180,21 @@ public void testMakeCellRenderContextThreeRows() {
         final BaseGridRow row2 = new BaseGridRow();
         final BaseGridRow row3 = new BaseGridRow();
         final List<GridColumn<?>> allColumns = new ArrayList<>();
+        final List<Double> allRowHeights = new ArrayList<>(Collections.nCopies(3, row1.getHeight()));
```

Comment:
ok but is the `new ArrayList()` wrapper really needed?

Reply:
No. You are correct. Corrected in new commit.